analysis/parser/RawTextParser/src/main/kotlin/de/maibornwolff/codecharta/parser/rawtextparser/RawTextParser.kt

```diff
@@ -92,11 +94,11 @@ class RawTextParser(
     }
 
     private fun getWriter(): Writer {
-        return if (outputFile != null) {
-            BufferedWriter(FileWriter(outputFile!!))
-        } else {
-            OutputStreamWriter(output)
+        if (!test) {
```

Comment:
Is this condition only for debugging?

Reply:
no this is for executing the parser dialog tests. previously the default condition was to write the output in terminal. now a default file is created. during the build it would not be convenient to generate all the default files (also many tests would need to be corrected) therefore we still write the output of .jsons during the build in terminal.

Reply:
I think we should not change code only to be able to run unit tests properly. Why has the default behavior changed?

Reply:
I can see your point, but I thought the requirement was if the user does not enter any input into the interactive terminal we still create a default cc.json file in the working directory.

Reply:
Ah okay. I think this is not included in the scope of this issue. Now it is a bit mixed. 
I would rather name the default something like this <INPUT-FILE-NAME>-output.cc.json. But ideally we should change this in a separate PR. Currently the default output name is handled in de.maibornwolff.codecharta.tools.interactiveparser.ParserDialogInterface#getOutputFileName. But it seems that not all parsers/importers are using this method.